app/services/user-service.js

```diff
@@ -0,0 +1,158 @@
+import Ember from "ember";
+import ApiBaseService from "./api-base-service";
+import _ from "lodash";
+
+export default ApiBaseService.extend({
+  store: Ember.inject.service(),
+  session: Ember.inject.service(),
+  printerService: Ember.inject.service(),
+
+  adminAppRoles: ["Reviewer", "Supervisor"],
+  stockAppRoles: [
+    "Stock administrator",
+    "Stock fulfilment",
+    "Order administrator",
+    "Order fulfilment"
+  ],
```

Comment:
In theory, if this mapping was derived from the API. You could write the whole thing without ever hardcoding any role names